rerun_py/rerun_sdk/rerun/recording_stream.py

```diff
@@ -291,41 +408,668 @@ def __del__(self):  # type: ignore[no-untyped-def]
+
+        A single `path` might be handled by more than one loader.
+
+        This method blocks until either at least one `DataLoader` starts
+        streaming data in or all of them fail.
+
+        See <https://www.rerun.io/docs/getting-started/data-in/open-any-file> for more information.
+
+        Parameters
+        ----------
+        file_path:
+            Path to the file that the `file_contents` belong to.
+
+        file_contents:
+            Contents to be logged.
+
+        entity_path_prefix:
+            What should the logged entity paths be prefixed with?
+
+        static:
+            If true, the components will be logged as static data.
+
+            Static data has no time associated with it, exists on all timelines, and unconditionally shadows
+            any temporal data of the same type.
+
+            Otherwise, the data will be timestamped automatically with `log_time` and `log_tick`.
+            Additional timelines set by [`rerun.set_time_sequence`][], [`rerun.set_time_seconds`][] or
+            [`rerun.set_time_nanos`][] will also be included.
+
+        """
+
+        from ._log import log_file_from_contents
+
+        log_file_from_contents(
+            file_path=file_path,
+            file_contents=file_contents,
+            entity_path_prefix=entity_path_prefix,
+            static=static,
+            recording=self,
+        )
+
+    def log_file_from_path(
+        self,
+        file_path: str | Path,
+        *,
+        entity_path_prefix: str | None = None,
+        static: bool = False,
+    ) -> None:
+        r"""
+        Logs the file at the given `path` using all `DataLoader`s available.
+
+        A single `path` might be handled by more than one loader.
+
+        This method blocks until either at least one `DataLoader` starts
+        streaming data in or all of them fail.
+
+        See <https://www.rerun.io/docs/getting-started/data-in/open-any-file> for more information.
+
+        Parameters
+        ----------
+        file_path:
+            Path to the file to be logged.
+
+        entity_path_prefix:
+            What should the logged entity paths be prefixed with?
+
+        static:
+            If true, the components will be logged as static data.
+
+            Static data has no time associated with it, exists on all timelines, and unconditionally shadows
+            any temporal data of the same type.
+
+            Otherwise, the data will be timestamped automatically with `log_time` and `log_tick`.
+            Additional timelines set by [`rerun.set_time_sequence`][], [`rerun.set_time_seconds`][] or
+            [`rerun.set_time_nanos`][] will also be included.
+
+        """
+
+        from ._log import log_file_from_path
+
+        log_file_from_path(
+            file_path=file_path,
+            entity_path_prefix=entity_path_prefix,
+            static=static,
+            recording=self,
+        )
```

Comment:
This is a lot of new code. It looks like it is all just forwarding calls - can we forward the docstring too o we don't need to maintain two sets of them?